src/main/java/in/projecteka/consentmanager/user/model/LockedUser.java

```diff
@@ -0,0 +1,19 @@
+package in.projecteka.consentmanager.user.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+
+@Builder
+@AllArgsConstructor
+@Getter
+public class LockedUser {
+    private final int invalidAttempts;
+    private final String patientId;
+    @Builder.Default
+    private final Boolean isLocked = false;
+    @Builder.Default
+    private final String lockedTime = null;
```

Comment:
don't you need to know when was last failure last attempt made (that need not be a locked attempt `lockedTime`)?

Reply:
If you have that you may not need lockedtime.

Reply:
I didn't get you, Basically lockedTime is the time when the user is locked, once he unlocks it'll be empty again

Reply:
Dude, if I my login attempt was failed yesterday, should you increase my failure count today if my current attempt is failed? Understood?

Reply:
Yea got it but My thought process is different. but anyways will get back to you